dev/how-to/maintain/backup-and-restore/br.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Use BR to Backup and Restore Cluster Data
+summary: Learn how to backup and restore the data of the TiDB cluster using BR.
+category: how-to
+---
+
+# Use BR to Backup and Restore Cluster Data
+
+Backup & Restore (BR) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with [`mydumper`/`loader`](/dev/how-to/maintain/backup-and-restore/mydumper-loader.md), BR is more suitable for scenarios of huge data volume. This document describes the command line, offers detailed use cases, best practices, usage restrictions, and introduces the working principle of BR.
+
+## Command-line description
+
+A `br` command consists of sub-commands, options and parameters. The sub-command is the characters without `-` or `--`. The option is the characters that start with `-` or `--`. The parameter is the characters that immediately follow and pass to the sub-command or the option.
+
+This is a complete `br` command:
+
+`br backup full --pd "${PDIP}:2379" -s "local:///tmp/backup"`
+
+In this command,
+
+* `backup` is the sub-command of `br`.
+* `full` is the sub-command of `backup`.
+* `-s` or `--storage` specifies the path in which the backup files are stored.
+* `"local:///tmp/backup"` is the parameter of `-s`. `/tmp/backup` is the path in the local disk in which the backup files are stored.
+* `--pd` is the Placement Driver (PD) service address.
+* `"${PDIP}:2379"` is the parameter of `--pd`.
+
+### Sub-commands
+
+A `br` command consists of multiple layers of sub-commands. Currently, BR has the following three sub-commands:
+
+* `br backup` is used to backup the data of the TiDB cluster.
+* `br restore` is used to restore the data of the TiDB cluster.
```

Comment:
Ditto for all similar expressions in this PR. A list with a colon will make it easier to read and understand.